spec/data_subjects/gsheet_file_spec.rb

```diff
@@ -0,0 +1,101 @@
+require 'remi_spec'
+
+describe Extractor::Gsheet do
+  let(:remote_path) { '' }
+  let(:credentials) {
+    {
+      :client_id     => 'some_client_id',
+      :access_token  => 'some_access_token',
+      :refresh_token => 'some_refresh_token',
+      :client_secret => 'some_client_secret',
+      :folder_id     => 'some_google_folder_id'
+    }
+  }
+
+  let(:gsheet_file) {
+    Extractor::Gsheet.new(
+      credentials: credentials,
+      remote_path: remote_path
```

Comment:
I wonder if it would make more sense to put `folder_id` as a parameter separate from the credentials?  Or are those tied together?